rusoto/credential/src/container.rs

```diff
@@ -74,6 +74,13 @@ impl ContainerProvider {
     }
 }
 
+/// Default: create a new provider with the given handle.
+impl Default for ContainerProvider {
```

Comment:
Adding default implementations is a question mark for me. On one hand it's following best practices, on the other hand it may go against the grain of https://github.com/rusoto/rusoto/issues/1323 .